.github/workflows/tests.yml

```diff
@@ -15,3 +15,5 @@ jobs:
           architecture: 'x64'
```

Comment:
This is line 1 (since Github won't let me comment there): Change the name to `E2E Runner Test`.

Reply:
@Showtim3

Reply:
@kevinlee12 Hey Sorry forgot to resolve this, it's done!

Reply:
Also change the line in line 5 to `check_e2e_tests_are_capture_in_ci`

Reply:
@Showtim3

Reply:
sorry, I misspelled, please change to `check_e2e_tests_are_capture_in_ci` (past tense)

Reply:
Hey @kevinlee12 Are you misspelling again? Do you want it to be ```check_e2e_tests_are_captured_in_ci```?

Reply:
oops, yes.

Reply:
@kevinlee12 Does it looks fine now?